playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -14,7 +14,7 @@
     aws_access_key: "{{ lookup('env', 'S3_ACCESS_KEY_ID') }}"
     aws_secret_key: "{{ lookup('env', 'S3_SECRET_ACCESS_KEY') }}"
     aws_bucket_name: "{{ aws_bucket | default(clusterid ~ '-docker') }}"
-    aws_bucket_region: "{{ aws_region | lookup('env', 'S3_REGION') | default('us-east-1') }}"
+    aws_bucket_region: "{{ aws_region | default(lookup('env', 'S3_REGION')) | default('us-east-1', true) }}"
```

Comment:
This should probably be:
```
aws_bucket_region: "{{ aws_region | default(lookup('env', 'S3_REGION') | default('us-east-1', true)) }}"
```

That way the default is applied to the lookup and not the result of the previous default.